### PR TITLE
zh: tidb cluster offline installation

### PR DESCRIPTION
**Pull request by weekface**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by ran-huang** [comment]
One more thing, are we sure that the offline-related docs don't need translation?

File: zh/deploy-on-general-kubernetes.md [hunk 1]
```diff
@@ -121,6 +121,74 @@ TiDB Operator 部署并配置完成后，可以通过下面命令部署 TiDB 集
     kubectl apply -f ${cluster_name} -n ${namespace}
     ```
 
+    如果服务器没有外网，需要在有外网的机器上将 TiDB 集群用到的 Docker 镜像下载下来并上传到服务器上，然后使用 `docker load` 将 Docker 镜像安装到服务器上。
+
+    部署 TiDB 集群可能用到的 Docker 镜像有，这里假设 TiDB 集群版本是 `v4.0.0`：
```
[the remark this answers is not in the document]
Comment (weekface) [reply zh/deploy-on-general-kubernetes.md]: fixed, ptal

File: zh/initialize-a-cluster.md [hunk 2]
```diff
@@ -72,3 +72,30 @@ kubectl apply -f ${cluster_name}/tidb-initializer.yaml --namespace=${namespace}
 ```
 
 以上命令会自动创建一个初始化的 Job，该 Job 会尝试利用提供的 secret 给 root 账号创建初始密码，并且创建其它账号和密码（如果指定了的话）。初始化完成后 Pod 状态会变成 Completed，之后通过 MySQL 客户端登录时需要指定这里设置的密码。
+
+如果服务器没有外网，需要在有外网的机器上将集群初始化用到的 Docker 镜像下载下来并上传到服务器上，然后使用 `docker load` 将 Docker 镜像安装到服务器上。
+
+集群初始化用到的 Docker 镜像有：
```
[the remark this answers is not in the document]
Comment (weekface) [reply zh/initialize-a-cluster.md]: fixed, ptal

**Comment by weekface** [comment]
> One more thing, are we sure that the offline-related docs don't need translation?

@DanielZhangQD PTAL

**Pull request merged**